tests/infra/consortium.py

```diff
@@ -213,7 +211,9 @@ def open_network(self, remote_node, pbft_open=False):
         return Calls:call("open_network")
         """
         proposal = self.get_any_active_member().propose(remote_node, script)
-        self.vote_using_majority(remote_node, proposal)
+        self.vote_using_majority(
+            remote_node, proposal, wait_for_global_commit=(not pbft_open)
```

Comment:
That doesn't seem like it should change because of the re-keying? I am missing something?

Reply:
I've reverted the change to the `pbft_open` flag. I was (wrongly) suspecting that we didn't need this flag anymore but we still do.

Reply:
I was missing something, ignore this.